test/Fail/Tests.hs

```diff
@@ -61,7 +61,7 @@ mkFailTest agdaFile =
   updGolden  = writeTextFile goldenFile
 
   doRun = do
-    let agdaArgs = ["-v0", "-i" ++ testDir, "-itest/" , agdaFile
+    let agdaArgs = ["-v0", "-vwarning:1", "-i" ++ testDir, "-itest/" , agdaFile
```

Comment:
Ah, this explains the additional warnings unrelated to `abstract`.

Reply:
I was mystified for a little bit as to why my new warnings didn't show up in the golden value.